src/controllers/requests/editRequests.js

```diff
@@ -0,0 +1,44 @@
+const db = require('../../database/db');
+require('dotenv').config();
+
+const getRequestsById = async (id) => {
+    const query = 'SELECT * from requests WHERE id = $1';
```

Comment:
You should query with both id and userId, so that you're sure that the user trying to edit is the owner